Sources/SwiftSyntaxBuilderGeneration/Tokens.swift.gyb

```diff
@@ -28,8 +28,6 @@ public class Token {
   public let text: String?
   public let classification: String
   public let isKeyword: Bool
-  public let requiresLeadingSpace: Bool
-  public let requiresTrailingSpace: Bool
```

Comment:
Since the `SyntaxFactory` gyb-template takes care of this, I have removed these (no longer needed) attributes from `Tokens`. Is this fine?

Reply:
Yes 👍